fhirpipe/utils.py

```diff
@@ -0,0 +1,64 @@
+import re
```

Comment:
can't these functions live in fhirpipe/extract/sql ? are they used somewhere else ?

Reply:
Some of them are used at several places, some others aren't... I can move the ones that are not in their respective files